### ISSUE TITLE
I'd like to help with the 3.0 release

### ISSUE THREAD

**Issue by nepthar** [mikebrady/shairport-sync#458]
Howdy all. I use this all the time on my raspberry pi and I've love to help out with the 3.0 release.

I've looked through the open issues & various docs, but I'm not sure what to focus on. Is there some guidance around what needs to be done for the 3.0 release?

Thanks!

**Comment by mikebrady** [comment]
Thanks for the offer. I'm sorry to say that I haven't thought too much about what needs to be done for 3.0. I'm traveling for another week or so and will give some thought to it. Off the top of my head, the only issue nagging at me is that it uses a little more CPU than 2.X. If that could be reduced, it would be great.

**Comment by mikebrady** [comment]
Just FYI – have a look at #472, if you happen to be GDbus guru...

**Comment by nepthar** [comment]
Oops, I thought I replied to this earlier. I'm not a GDbus guru, but I've actually started on a small project - adding additional remote commands.

Looks like there's already a pause command: https://github.com/mikebrady/shairport-sync/blob/master/rtp.h#L19

Also, it seems like your Dbus integration work will probably use these types of commands, so before I jump in, have you started progress on this?
